apps/app_rpt.c

```diff
@@ -6232,6 +6236,9 @@ static int rpt_exec(struct ast_channel *chan, const char *data)
 		if (b1)
 			ast_shrink_phone_number(b1);
 		cfg = ast_config_load("rpt.conf", config_flags);
+		if (cfg == CONFIG_STATUS_FILEINVALID) {
```

Comment:
Shouldn't there also be a warning here? If this happens, the config was valid at some point but is now invalid, which seems pretty bad

Reply:
Here, I'm not concerned.  No specific warning was being logged if the file was removed so I took the same approach if the configuration was modified and is no longer valid.  But, regardless, you will get the "Cannot find specified system node %s\n" log message issued at the end of the function.